templates/message.html

```diff
@@ -0,0 +1,349 @@
KoUCSUEKSJBOhQREaFTyVBCKUVCBnnUDEoJVTqEDFlFBYcMBmcpwed3ZyKxNPDju3d3//fu/30X3Uca/MllQBPVR013seklMCevUPNLKHpvsaG2sKmnkScpWcUq1x6pQ6T1ayypHnZ1E1c5t04+kSJZJgnykjwja+Sxq+N+smDHI8wy5mNXWNcGPEFHfqHi7VEfED57J3w+QkdzrE6jjNi1KXT8HteOqQEqUh8q98kS33sTE6x5L1UgdgMzkmOf82SHfbxA3Z6ZOsPv3zXZaCBVtSUprEgLoWmjTC1LAWV9D4EkMS2HCFUH+6oTVc1XNw5jlxDaeWm4/aGtMQHrm1jUdXpyiAPzhj2d4JoZYM6OTYPf79LHRTWgZmz/Q+/3OO6SHHnq/OMeuc6equqOv48C35kzJeRdDb23c4Lo1KzhuZtrYJbEXS/HCL0Uis7vGhKcX9DfcZv1Ga+PW+QiidP7rPX9PPxS9Ntm4XIYQ0t0xCw+UL9RPa+J5CiHs/BcZas2i3FcFsxMDuib9f0c/BZWXBaF/2EGX+j/Q+pH8kO2UfqXw1nsHbPKLMaxWbisqbEkKrFt7rFnqqoEmTLvOL/OvIaqN5jRZ5L6C35Sn1Dvc41ZjBAgf4GoOAJHi/8T3iGLAdJGo0e/dm2tnuRdmETGvpfnzHptTLBvyCzvHPF3ECD4A/DL6HUAAAB42mNgYNCBwgKGRYxtTGJMG5idmBOYu5i3MN9jUWBxYklg6WBZxvKNVYu1ivUNmx3bKnYpdjv2fRxmHDkcKzj2cNzj+MSZxMXCZcO1hOsDtwF3DvciHgEeD540nik8p3g+8DrwtvDe4ZPja+K7we/AX8C/hv+awAVBJkEDwTjBLsEVglcEPwlxCZkILRHmEw4S3iUiJ1IkqiVaJrpO9IeYkZiP2CaxH+Ju4iskOCQ8JNZI8kgGSdZIGUmlSK2QuiZtJO0jvUL6mfQzGRWZDpl3simyJ2RfybXJHZIPkz8m/0JBQEFP4ZrCO0UzxTbFK0pmSglKm5S+KEsoeykXKe9QvqMSp/JIdZGalFqC2jK1T+ol6rc0fDQWaCpp5mhu02LTytGapXVKm0HbTXuVjp7ONJ1/ujl6LHodeqv0VfSL9A8YiBjUGOwyZDKcZiRmNM1YwfiGSYOpiekjsyizM+YO5jssXCyKLOZZXLGUseyxErIqsLpgbWJ9zMbJpsnmm62P7RI7BrsyuxP2WvZzHMQcNjlyOdo49uCAsxxXOG5zPOf4zInHycApxGma0zVnMWcX5wlAuMX5lvMtlySXYy7vXHNcr7lNc7cBAFbGk3YAAAABAAAA6gBBAAUAAAAAAAIAAQACABYAAAEAAYgAAAAAeNqdU01PU0EUPa9TVD5slBBDDDFvyYI+X5tgDDtEPjRVEyDqws3jtdRiS/F1CMGf4Mpf4MqF/AV+AcLOnX+AFWvWnLlzW6hUTUxzp2fu3HvmnnvnAZjAOQyC/DCAjOZxgAfceZxDAV8UG6zgq+I8IpwqHsJkMKH4BqaCGcU38S1YUXwL08GZ4lE8znXjx/AuV1F8m/i74gLWTV7xHYybt4rvYsRsKT7CPWMV/0BsPis+RsEcKj5h/JHHvwzum594igbqNEv7hBqqCGkJ9wlRijZ2sE/9Luo9vSEOaGXEKNGKikqYoXeZ0W3GNckTYoE4Y7ZbE+FvY5t9ekVfjSjEGv3b6BBVevyrPKtjlxwJ8+Z5mkp0lWvGyCLt7wwhnjC/QQanxVUZ/zOj/9bXcldHK3YckfB0WbocxSscg25oyOr6aKUHTkNLbvhAXxub13qWiNJQovb5vyHeTKpzbFYq81NqyG2peNy0/H6LKjKJrXJNe33vUMP1bg6ekdNj6Z3DQ/725BfxvD871dxIUIuR/5tnqXVHVNWk63XG+glEwtlidyqipiZKvP7dKzos41yn5smTMM7v+nPcC/19smXeEP+x7kuuSGqu87TZx9mhp4Jn7OMiXnLyi/JFRANf3OVbecOTDU7f1WD1fcV4oTU9F2XOOytriXWVZY3xqPftzYr2TXK7V2ulPq9qqce7ho/yJWTyZpoXuXPFcgAAeNpt0DdsU3EQx/HvJY6dOL330Hvze7ZT6HaSR++9E0hcICTBwUBoAdGrQEgwgWgLIHoVCBgA0ZsoAgZmuhiAFZy8Pxu3fHQn3en0I4rW+uPDx//qM0iURBONhRis2IglDjvxJJBIEsmkkEoa6WSQSRbZ5JBLHvkUUEgRbWhLO9rTgY50ojNd6Eo3utODnvSiN31woKHjxIWbYkoopYy+9KM/AxjIIAbjwUs5FVRiMIShDGM4IxjJKEYzhrGMYzwTmMgkJjOFqUxjOjOYySxmM4e5VImFo2xgIzfYx0c2sYvtHOA4xySGbbxnPXvFKjZ2sp8t3OaDxHKQE/ziJ785wikecI/TzGM+u6nmETXc5yHPeMwTnvIpkt5LnvOCM/j5wR7e8IrXBPjCN7aygCALWUQtdRyinsU0EKKRMEtYyrJIystZQRMrWc0qrnKYZtawlnV85TvXOMs5rvOWdxIndomXBEmUJEmWFEmVNEmXDMmULM5zgctc4Q4XucRdNnNSsrnJLcmRXHZInuRLgRRKkdVf29QQ0GzhuqDD4agw9TiUqvfqSqeyrEU9sqDUlLrSqXQp3cpiZYmyVPnvnsdUU3c1ze4L+sOhmuqqxoA50g1Tt2GpDIfqWxu3Ud6i4TX/iKgrnUrXXwhgnS8AeNo9zb0OwVAYxvEebU8/9LtlIzU0xEmEGzBply4iEm3iOswWI27lrcnuwnjCcbbn9yz/J3tfiF21muxt0zF2a7uKi2ZCUVtTtsM4tyPi4thopOcl6WJDRl4+9GFPfGECxh8cMF8SFsDvEjZgHSQcwF5LuIBTSPQBd/EDI08Gfbzesic6vTqBAeiPFUMw2CtGYLhSjMForpiA8UwxBZOpYgamheIAzBRbysQHzsRVBgAAAVV+7WIAAA==) format('woff');
```

Comment:
I didn't notice this was for the email templates. Try it out and let me know, but if not it's fine.